packages/orbit-components/src/tmp-tailwind/SegmentedSwitch/SwitchSegment/index.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import cx from "clsx";
+
+import useClickOutside from "../../../hooks/useClickOutside";
+import type { Props } from "../../../SegmentedSwitch/SwitchSegment/types";
+
+const SwitchSegment = ({
+  value,
+  label,
+  disabled,
+  onChange,
+  setTooltipShown,
+  onFocus,
+  defaultChecked,
+  name,
+}: Props) => {
+  const ref = React.useRef<HTMLInputElement | null>(null);
+  useClickOutside(ref, () => setTooltipShown(false));
+
+  return (
+    <label className="orbit-switch-segment-label relative flex w-full">
+      <input
+        className={cx(
+          "absolute m-[-1px] h-[1px] w-[1px] appearance-none overflow-hidden whitespace-nowrap border-0 p-0 focus:outline-none",
+          "peer",
+        )}
+        name={name || "switch-segment"}
+        defaultChecked={defaultChecked}
+        type="radio"
+        role="switch"
+        ref={ref}
+        aria-checked={Boolean(value)}
+        onChange={onChange}
+        onFocus={onFocus}
+        disabled={disabled}
+        value={value}
+      />
+      <div
+        className={cx(
+          "orbit-switch-segment-text",
+          "peer-checked:text-form-element-label-filled-foreground peer-checked:outline-blue-normal peer-checked:z-10 peer-checked:!rounded-[5px] peer-checked:outline peer-checked:outline-2 ",
+          "peer-focus:text-form-element-label-filled-foreground peer-focus:outline-blue-normal peer-focus:z-10 peer-focus:!rounded-[5px] peer-focus:outline peer-focus:outline-2 ",
+          "p-sm font-base text-normal box-border flex w-full max-w-full cursor-pointer items-center justify-center border-0 text-center font-medium leading-normal transition-colors duration-150 ease-in-out",
+          "bg-form-element-background text-form-element-foreground hover:text-form-element-label-filled-foreground",
+          "shadow-switch-segment",
```

Comment:
I think here will be better just to use it as custom, no need to add that to a preset, when it will be used only once here in SegmentSwitch :)

Reply:
I'm not sure if I understand what you mean. Do you mean having something like `shadow-[...]`?

Reply:
And how can I inject the color token there (`${theme.orbit.paletteCloudDark}`)? By hardcoding it?

https://github.com/kiwicom/orbit/blob/a4c52fad33ea8d85ef6498c3a6156ebeef9d8e5a/packages/orbit-components/src/SegmentedSwitch/SwitchSegment/index.tsx#L35

Reply:
I think this should work 🤔 :

```
 style={{boxShadow: `0 0 0 1px ${theme.orbit.paletteCloudDark}`}} 
```

Reply:
It works, but using `defaultTheme` instead, is that alright?

Reply:
with [`useTheme` hook](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/hooks/useTheme/index.ts) ;)